lib/src/argument_list_visitor.dart

```diff
@@ -447,7 +441,7 @@ class ArgumentSublist {
       rule.disableSplitOnInnerRules();
 
       // Tell it to use the rule we've already created.
-      visitor.beforeBlock(_blocks[argument], blockRule, previousSplit);
+      visitor.beforeBlock(_blocks[argument]!, blockRule!, previousSplit);
```

Comment:
To avoid the first `!` on this line, consider doing:
```dart
var argumentBlock = _blocks[argument];
if (argumentBlock != null) {
  rule.disableSplitOnInnerRules();

  // Tell it to use the rule we've already created.
  visitor.beforeBlock(argumentBlock, blockRule!, previousSplit);
  ...
}
```